src/hotspot/share/runtime/globals.hpp

```diff
@@ -865,10 +865,16 @@ const intx ObjectAlignmentInBytes = 8;
           "JVM exits on the first occurrence of an out-of-memory error "    \
           "thrown from JVM")                                                \
                                                                             \
+  /* SapMachine 2021-05-21: Changed comment (we do not core on OOM) */      \
   product(bool, CrashOnOutOfMemoryError, false,                             \
-          "JVM aborts, producing an error log and core/mini dump, on the "  \
+          "JVM aborts, producing an error log, on the "                     \
```

Comment:
Should the possibility to explicitly enable core/mini dump and the printing of the thread stack be mentioned here?